modules/3d/include/opencv2/3d/ptcloud.hpp

```diff
@@ -8,6 +8,8 @@
 #ifndef OPENCV_3D_PTCLOUD_HPP
 #define OPENCV_3D_PTCLOUD_HPP
 
+#include "opencv2/flann.hpp"
```

Comment:
Do we need this include as part of the **public** API? Could we move it to implementation-only?

Reply:
Sure, this include should be moved to src folder since all flann-dependent function arguments were removed or replaced.

Reply:
Sorry, I forgot to remove it, now the flann dependency has been removed.